lms/resources/_js_config.py

```diff
@@ -337,9 +337,13 @@ def _auth_token(self):
         """Return the authToken setting."""
         return BearerTokenSchema(self._request).authorization_param(self._lti_user)
 
-    def _blackboard_files_available(self):
-        """Return True if the Blackboard Files API is available to this request."""
-        return self._request.feature("blackboard_files")
+    def _blackboard_files_enabled(self):
+        """
+        Return True if the Blackboard Files API is enabled for this request.
+
+        :raise ConsumerKeyError: if request.lti_user.oauth_consumer_key isn't in the DB
+        """
+        return self._application_instance().settings.get("blackboard", "files_enabled")
```

Comment:
I think this might be better in the body of the method it's used in. I think it's simple enough and possibly clearer to see where the settings are coming from inline. Maybe something like:

```python
setting = self._application_instance().setting
...
"blackboard: {
    "enabled": settings.get("blackboard", "files_enabled"),
    ...
}
```

_Irrelevant:_ Also... I still want that google inner function dead...